pointmatcher/DataPointsFilters/utils/utils.h

```diff
@@ -39,7 +39,18 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <vector>
 #include <algorithm>
 
-namespace utils {
+namespace PointMatcherSupport
+{
+
+template<class T>
+inline constexpr T pow_(const T base, const std::size_t exponent)
+{
+    return (exponent == 0 ? 1 : base * pow_(base, exponent - 1));
+}
+/* use template to force compile time evaluation : https://stackoverflow.com/a/16443849 */
```

Comment:
You really don't need that :). 
First, it would be wrong to optimize away the few microseconds on program load (they are no worth the extra complication).  
Second, the way you use pow forces compile time anyway, since you use `nbCells` in context that require compile time evaluation, e.g. here https://github.com/ethz-asl/libpointmatcher/pull/255/files#diff-0e2e58e13d0edca69e786bb76e564ac1R101.